test/config/database_global.json

```diff
@@ -5,11 +5,11 @@
         },
         {
             "namespace" : "asic0",
-            "include" : "database_config.json"
+            "include" : "/sonic/src/sonic-py-swsssdk/test/config/database_config.json"
```

Comment:
>/sonic/src/sonic-py-swsssdk/test/config/ [](start = 25, length = 40)

The test may be run outside slave docker, so the absolute path may be invalid. Could you use relative path?

Reply:
Updated.